pyi.py

```diff
@@ -42,6 +42,58 @@ class TypeVarInfo(NamedTuple):
     name: str
 
 
+# The collections import blacklist is for typing & typing_extensions
+#
+# OrderedDict is omitted:
+# -- In Python 3, we'd rather import it from collections, not typing or typing_extensions
+# -- But in Python 2, it cannot be imported from collections or typing, only from typing_extensions
+#
+# ChainMap does not exist in typing or typing_extensions in Python 2,
+# so we can disallow importing it from anywhere except collections
+_BAD_COLLECTIONS_ALIASES = {
```

Comment:
These names are kind of unclear, could you name them similarly to _TYPING_NOT_TYPING_EXTENSIONS with both the module they should be imported from and the module they should *not* be imported from?

Reply:
sure

Reply:
Done.

Reply:
Thanks!